components/regimes/es/ticketbai.templ

```diff
@@ -2,17 +2,26 @@ package es
 
 import (
 	"github.com/invopop/gobl"
-	"github.com/invopop/gobl/addons/es/tbai"
 	"github.com/invopop/gobl.html/components/images"
+	"github.com/invopop/gobl/addons/es/tbai"
 )
 
 // TicketBAIQR generates a QR code area for the TicketBAI code in the given envelope.
 templ TicketBAIQR(env *gobl.Envelope) {
+	if result, code, qr := HasTicketBAIQR(env); result {
+		@generateTicketBAIQR(code, qr)
+	}
+}
+
+// HasTicketBAIQR returns a boolean indicating whether the envelope has a TicketBAI QR or not.
+func HasTicketBAIQR(env *gobl.Envelope) (bool, string, string) {
```

Comment:
These method names and comments don't match the output, which I don't think is a good DX. It'd probably be better to return `nil` or an object to be used later.